### PR TITLE
Improve Endpoint page

### PR DESCRIPTION
**Pull request by dacolgit**

### CODE REVIEW

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerEndpointContent.razor [hunk 1]
```diff
@@ -0,0 +1,82 @@
+﻿@*------------------------------------------------------------
+     Copyright (c) Microsoft Corporation.  All rights reserved.
+     Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+    ------------------------------------------------------------*@
```
Comment (luiscantero) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerEndpointContent.razor:4]: This indentation is expected?

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Endpoints.razor [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿@*------------------------------------------------------------
-  Copyright (c) Microsoft Corporation.  All rights reserved.
-  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
-------------------------------------------------------------*@
+      Copyright (c) Microsoft Corporation.  All rights reserved.
+      Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+    ------------------------------------------------------------*@
```
Comment (luiscantero) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Endpoints.razor:4]: This indentation is expected?

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Endpoints.razor [hunk 3]
```diff
@@ -72,39 +69,31 @@
...
-                    </div>
-                </td>
-                <td class="hover-text width-small">
+             endpoint.EndpointModel.EndpointState == EndpointConnectivityState.Unauthorized) {
+            <td class="hover-text width-large">
+                <div>
+                    @{ string discover = DiscovererId ?? RegistryHelper.PathAll; }
+                    @{ string application = ApplicationId ?? RegistryHelper.PathAll; }
+                    @{ string supervisor = SupervisorId ?? RegistryHelper.PathAll; }
+                    <a href='browser/1/@discover/@application/@supervisor/@endpoint.EndpointModel.Registration.Id'>
```
Comment (luiscantero) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Endpoints.razor:81]: Why not just one @{ ... } block for the three lines?

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerEndpointContent.razor (hunk 1 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerEndpointContent.razor:4]: no fixed

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Endpoints.razor (hunk 3 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Endpoints.razor:81]: no fixed

**Pull request merged**